node/routes.py

```diff
@@ -86,8 +85,7 @@ def post_dispute() -> Response:
 
     if req_data["sequencer_id"] != zconfig.SEQUENCER["id"]:
         return error_response(ErrorCodes.INVALID_SEQUENCER)
-
-    if zdb.has_missed_txs():
+    if zdb.has_missed_txs() or (req_data['app_name'] == '' and zdb.is_sequencer_down):
```

Comment:
what does empty app name mean?

Reply:
Yes, an empty app name with an empty list of txs does not get stored in the database.